src/app/shared/ui/paragraphs/paragraphs.component.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Component for generating paragraphs by splitting lines of text
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+import { Component, input } from '@angular/core';
+import { SplitLinesPipe } from '@app/shared/pipes/split-lines.pipe';
+
+/**
+ * Component for generating paragraphs by splitting lines of text
+ */
+@Component({
+  selector: 'app-paragraphs',
+  imports: [SplitLinesPipe],
+  templateUrl: './paragraphs.component.html',
+})
+export class ParagraphsComponent {
+  text = input.required<string>();
```

Comment:
Would be nice if the text could be taken from the text content of the tag, but there's no simply way to do it. There is only a simply way to project it as-is into the template. So let's keep it as an attribute.